modules/connectivity/locals.tf

```diff
@@ -831,13 +832,31 @@ locals {
     }
     "${local.root_id}-corp" = {
       parameters = {
-        Enable-DDoS-VNET = {
-          ddosPlan = local.ddos_protection_plan_resource_id
+        Deploy-Private-DNS-Zones = {
```

Comment:
Are there some missing from here like Backup?

Reply:
Corp currently only implements `Deny-Public-Endpoints` and `Deploy-Private-DNS-Zones` Policy Assignments, of which only `Deploy-Private-DNS-Zones` has anything we configure using the config.

The configured items are based on where we had settings in the Portal experience for ARM.

This may have since been updated, and we may want to review this more broadly though?